### PR TITLE
Individual section for API Testing

### PR DESCRIPTION
**Pull request by orviz**

### CODE REVIEW

File: content/06.quality_criteria.md [hunk 1]
```diff
@@ -33,13 +66,19 @@ agreed design specifications. This type of **Service** testing focus on the eval
...
 
-* **[SvcQC.Fun01]** Functional testing SHOULD tend to cover the full set of functionality that
-  the **Service** claims to provide. [SQA-QC.Fun01].
-  * **[SvcQC.Fun01.1]** Functional tests SHOULD be checked automatically.
-  * **[SvcQC.Fun01.2]** Functional tests MUST include the public APIs of the **Service**.
-  * **[SvcQC.Fun01.3]** Functional tests SHOULD include the Web interface of the **Service**.
+* **[SvcQC.Fun01]** Functional testing SHOULD tend to cover the full scope 
+--e.g. positive, negative, edge cases-- for the set of functionality that the
+**Service** claims to provide. [SQA-QC.Fun01].
+  * **[SvcQC.Fun01.1]** Functional tests MUST include the public APIs of the
```
Comment (mariojmdavid) [inline content/06.quality_criteria.md:72]: seems to duplicate [SvcQC.Api01] the criteria is already a "MUST", 
I know that API testing is part of the Functional testing but in practice it is already on its own section, in that section maybe say that API testing is a subset of Fuctional testing?
if need be add any aditional text to [SvcQC.Api01] , i.e.

When using APIs, contract testing MUST detect any disruption in
  the features exposed by the provider to the consumer, through the validation
  of the API specification

Comment (jorge-lip) [reply content/06.quality_criteria.md:72]: Not a complete duplication.  [SvcQC.Api01] is specifically focus on the OpenAPI specification, while [SvcQC.Fun01.1] is agnostic in terms of the API specification which might not even be RESTful.

Comment (mariojmdavid) [reply content/06.quality_criteria.md:72]: ok, agree

Comment (orviz) [reply content/06.quality_criteria.md:72]: Just want to highlight the fact that functional testing should cover the API features that are relevant to the end user. 

+1 on leaving the last sentence & removing the first one

**Comment by mariojmdavid** [comment]
@jorge-lip if you agree, approve the PR
